### ISSUE TITLE
x265 only builds 8 bit

### ISSUE THREAD

**Issue by hyhy01** [markus-perl/ffmpeg-build-script#101]
input
```sh
ffmpeg -y -i "$in_file" -c:v libx265 -tag:v hvc1 -deblock 1:1 -x265-params profile=main10:level-idc=51:no-amp=1:limit-tu=4:aq-strength=0.4:aq-mode=3  -preset veryslow -tune animation -crf 30 -acodec libfdk_aac -profile:a aac_he_v2 -vbr 4 -f mp4 "$out_file" 
# or
ffmpeg -y -i "$in_file" -c:v libx265 -tag:v hvc1 -profile:v high -level 5.0  -pix_fmt yuv420p10 -deblock 1:1 -x265-params no-amp=1:limit-tu=4:aq-strength=0.4:aq-mode=3
```
output
```
ffmpeg version 4.4 Copyright (c) 2000-2021 the FFmpeg developers
  built with gcc 9 (Ubuntu 9.3.0-17ubuntu1~20.04)
..............
  Stream #0:0: Audio: cook (cook / 0x6B6F6F63), 44100 Hz, stereo, fltp, 64 kb/s
  Stream #0:1: Video: rv40 (RV40 / 0x30345652), yuv420p, 640x480, 470 kb/s, 24 fps, 24 tbr, 1k tbn, 1k tbc
Codec AVOption deblock (Loop filter parameters, in <alpha:beta> form.) specified for output file #0 (xxxxx.mp4) has not been used for any stream. The most likely reason is either wrong type (e.g. a video option with no video streams) or that it is a private option of some encoder which was not actually used for any stream.
Stream mapping:
  Stream #0:1 -> #0:0 (rv40 (native) -> hevc (libx265))
  Stream #0:0 -> #0:1 (cook (native) -> aac (libfdk_aac))
Press [q] to stop, [?] for help
Incompatible pixel format 'yuv420p10le' for codec 'libx265', auto-selecting format 'yuv420p'
x265 [info]: HEVC encoder version unknown
x265 [info]: build info [Linux][GCC 9.3.0][64 bit] 8bit
x265 [info]: using cpu capabilities: MMX2 SSE2Fast LZCNT SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2
x265 [info]: Main profile, Level-3 (Main tier)
x265 [info]: Thread pool created using 12 threads
x265 [info]: Slices                              : 1
x265 [info]: frame threads / pool features       : 3 / wpp(8 rows)
x265 [info]: Coding QT: max CU size, min CU size : 64 / 8
x265 [info]: Residual QT: max TU size, max depth : 32 / 3 inter / 3 intra
x265 [info]: ME / range / subpel / merge         : star / 57 / 4 / 5
x265 [info]: Keyframe min / max / scenecut / bias  : 24 / 250 / 40 / 5.00 
x265 [info]: Lookahead / bframes / badapt        : 40 / 10 / 2
x265 [info]: b-pyramid / weightp / weightb       : 1 / 1 / 1
x265 [info]: References / ref-limit  cu / depth  : 5 / off / off
x265 [info]: AQ: mode / str / qg-size / cu-tree  : 3 / 0.4 / 32 / 1
x265 [info]: Rate Control / qCompress            : CRF-30.0 / 0.60
x265 [info]: tools: rect rd=6 psy-rd=0.40 rdoq=2 psy-rdoq=1.00 rskip mode=1
x265 [info]: tools: limit-tu=4 signhide tmvp b-intra strong-intra-smoothing
x265 [info]: tools: deblock(tC=1:B=1) sao
[libfdk_aac @ 0x55bc51677980] Note, the VBR setting is unsupported and only works with some parameter combinations
Output #0, mp4, to 'xxxx.mp4':
  Metadata:
    title           : xxxxx
    copyright       : (c)2002 RealMedia
    ASMRuleBook     : #($Bandwidth >= 0),Stream0Bandwidth = 64082, Stream1Bandwidth = 465918;
    Audiences       : BatchRealProducer's Audience FreeEdition;
    audioMode       : music
    Creation Date   : 6/13/2004 23:34:22
    Description     : This RealMedia File Compressed by Batch Real Producer
    Generated By    : Helix DNA Producer SDK 9.2 for Windows, Build 9.2.0.459
    Modification Date: 6/13/2004 23:34:22
    videoMode       : sharp
    encoder         : Lavf58.76.100
  Stream #0:0: Video: hevc (hvc1 / 0x31637668), yuv420p(progressive), 640x480, q=2-31, 24 fps, 12288 tbn
    Metadata:
      encoder         : Lavc58.134.100 libx265
    Side data:
      cpb: bitrate max/min/avg: 0/0/0 buffer size: 0 vbv_delay: N/A
  Stream #0:1: Audio: aac (mp4a / 0x6134706D), 44100 Hz, stereo, s16
    Metadata:
      encoder         : Lavc58.134.100 libfdk_aac
```
yuv420p10 not work

**Comment by paladine** [comment]
Addressed by PR #102

**Comment by markus-perl** [comment]
Implemented with https://github.com/markus-perl/ffmpeg-build-script/pull/106
